WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -111,6 +112,28 @@ public function register() {
 		];
 	}
 
+	/**
+	 * Reset short echo context tracking variable for a new file.
+	 *
+	 * @since 2.3.4
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+	 * @param int                         $stackPtr  The position of the current token
+	 *                                               in the stack passed in $tokens.
+	 *
+	 * @return int|void Integer stack pointer to skip forward or void to continue
+	 *                  normal file processing.
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		static $current_file;
+		if ( $phpcsFile !== $current_file ) {
+			$this->in_short_echo = false;
+			$current_file        = $phpcsFile;
```

Comment:
I'd suggest comparing the actual filename instead of comparing two `File` objects, i.e. using `$phpcsFile->getFilename()`.

Reply:
Oh, I meant to do that - I forgot to check if `getFilename()` returned just the filename or the full path - according to [this](https://github.com/squizlabs/PHP_CodeSniffer/blob/2596a15760a1e652d64707d5ca21af0a31e5d2d8/src/Files/File.php#L1214), it's the absolute path.

Reply:
Handled in [`f1cd93c` (#748)](https://github.com/Automattic/VIP-Coding-Standards/pull/748/commits/f1cd93c77c7a3dcb70348897f2bcce835e161763)